docs/monitor/metrics.md

```diff
@@ -79,6 +79,7 @@ Metrics Prefix | Metrics Suffix | Type | Since | Description
 `kyuubi.backend_service.fetch_result_rows_rate`  | | meter | 1.5.0 |<div style='width: 150pt;word-wrap: break-word;white-space: normal'> kyuubi backend service `fetchResults` method that fetch result rows rate </div>
 `kyuubi.backend_service.get_primary_keys`  | | meter | 1.6.0 |<div style='width: 150pt;word-wrap: break-word;white-space: normal'> kyuubi backend service `get_primary_keys` method execution time and rate </div>
 `kyuubi.backend_service.get_cross_reference`  | | meter | 1.6.0 |<div style='width: 150pt;word-wrap: break-word;white-space: normal'> kyuubi backend service `get_cross_reference` method execution time and rate </div>
+`kyuubi.operation.state`   | `${operationType}`<br/>`.${state}`     | counter | 1.6.0 |<div style='width: 150pt;word-wrap: break-word;white-space: normal'>  current count for the `${operationType}` with a particular `${state}`, e.g. `BatchJobSubmission.pending`</div>
```

Comment:
you can use meters for this actually.. just like kyuubi.operation.state.$state

```
"kyuubi.operation.state.closed" : {
      "count" : 19,
      "m15_rate" : 0.020137396109391522,
      "m1_rate" : 1.6595792398582304E-4,
      "m5_rate" : 0.003290775624396568,
      "mean_rate" : 0.00539128492410111,
      "units" : "events/second"
    },
    "kyuubi.operation.state.error" : {
      "count" : 1,
      "m15_rate" : 0.025462486603774714,
      "m1_rate" : 7.483284619160424E-15,
      "m5_rate" : 4.1270758361306424E-4,
      "mean_rate" : 5.372806924825531E-4,
      "units" : "events/second"
    },
    "kyuubi.operation.state.finished" : {
      "count" : 18,
      "m15_rate" : 0.01815503925250486,
      "m1_rate" : 1.5388408901182632E-14,
      "m5_rate" : 1.1365883986004687E-4,
      "mean_rate" : 0.005107080000702363,
      "units" : "events/second"
    },
    "kyuubi.operation.state.initialized" : {
      "count" : 19,
      "m15_rate" : 0.006237933279653059,
      "m1_rate" : 1.5785160613558252E-14,
      "m5_rate" : 1.1526188920321656E-4,
      "mean_rate" : 0.005376601901198222,
      "units" : "events/second"
    }
```

Reply:
thanks, let me have a try

Reply:
thanks, it helps a lot.

Reply:
the discussion is marked as resolved but the code part remains unchanged？

Reply:
...resolved locally.